### ISSUE TITLE
Version 1.10, resource properties missing

### ISSUE THREAD

**Issue by bjorndarri** [karakun/OpenWebStart#587]
Resource properties are not forwarded to the application since updating to 1.10.

Looks like all my applications are broken with this update, since they all have jdbc urls specified in the resource section.

I'm working with very simple jnlp files, here's an example:

```
<?xml version="1.0" encoding="utf-8"?>
<jnlp spec="1.0+" codebase="https://xxxxx.yyy/app/" href="application.jnlp">

  <information>
    <offline-allowed/>
  </information>

  <security>
    <all-permissions/>
  </security>

  <resources>
    <j2se version="11+"/>
    <jar href="application.jar"/>
    <property name="db.url" value="jdbc:oracle:thin:@xxxxx.yyyy.zz:1234:rrrr"/>
  </resources>
  
  <application-desc main-class="xx.xx.xx.ApplicationPanel"/>
</jnlp>
```

The system property `db.url` is no longer forwarded to the application. There's nothing in the log file, by the way.

I looked through recent commits and found an interesting line in this one: cce96c6

OwsJvmLauncher:250

```text
env.put(ApplicationInstance.IGNORE_JNLP_RESOURCE_PROPERTIES, "true");
```

I haven't looked into this any further, but this line does look a bit suspect, given the problem I'm running into :-).

**Comment by fleminra** [comment]
+1

All of our Web Start applications that depend on JNLP-set properties are fatally broken in OWS 1.10.0.

1.9.1 works.

**Comment by patrice0** [comment]
I've had the same problem. Here's a 2 line program that exhibits the problem:
import java.io.*;

```
public class Test
{
	public static void main(String args[])
	{
		ClassLoader cl = Test.class.getClassLoader();
		System.out.println(cl.getResourceAsStream(System.getProperty("filename")));
	}
}
```

**Comment by patrice0** [comment]
I've had the same problem.

**Comment by sclassen** [comment]
We found the source of the problem an plan on releasing a bugfix early next week

**Comment by patrice0** [comment]
Thank you!

**Comment by sclassen** [comment]
Fixed with [1.10.1](https://github.com/karakun/OpenWebStart/releases/tag/v1.10.1)
